---a/net/minecraft/network/play/server/S43PacketCamera.java
+++b/net/minecraft/network/play/server/S43PacketCamera.java
@@ -1,44 +1,39 @@
 package net.minecraft.network.play.server;
 
 import java.io.IOException;
+
 import net.minecraft.entity.Entity;
 import net.minecraft.network.INetHandler;
 import net.minecraft.network.Packet;
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
-import net.minecraft.world.World;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class S43PacketCamera implements Packet
-{
+
+public class S43PacketCamera implements Packet {
+
     public int field_179781_a;
     private static final String __OBFID = "CL_00002289";
 
-    public S43PacketCamera() {}
+    public S43PacketCamera() {
+    }
 
-    public S43PacketCamera(Entity p_i45960_1_)
-    {
+    public S43PacketCamera(Entity p_i45960_1_) {
         this.field_179781_a = p_i45960_1_.getEntityId();
     }
 
-    public void readPacketData(PacketBuffer buf) throws IOException
-    {
+    public void readPacketData(PacketBuffer buf) throws IOException {
         this.field_179781_a = buf.readVarIntFromBuffer();
     }
 
-    public void writePacketData(PacketBuffer buf) throws IOException
-    {
+    public void writePacketData(PacketBuffer buf) throws IOException {
         buf.writeVarIntToBuffer(this.field_179781_a);
     }
 
-    public void func_179779_a(INetHandlerPlayClient p_179779_1_)
-    {
+    public void func_179779_a(INetHandlerPlayClient p_179779_1_) {
         p_179779_1_.handleCamera(this);
     }
 
-    @SideOnly(Side.CLIENT)
-    public Entity func_179780_a(World worldIn)
-    {
-        return worldIn.getEntityByID(this.field_179781_a);
+    public void processPacket(INetHandler handler) {
+        this.func_179779_a((INetHandlerPlayClient) handler);
     }
-
-    public void processPacket(INetHandler handler)
-    {
-        this.func_179779_a((INetHandlerPlayClient)handler);
-    }
-}
+}
